src/main/react-front-end/src/views/Support/SupportComponent.js

```diff
@@ -95,7 +96,8 @@ export default class SupportComponent extends Component{
               <TextField
                 required
                 label = 'Email Address'
-                name = 'email' 
+                name = 'email'
+                value = { cookies.get('email') }
```

Comment:
Can you get it from store.getState().email instead of cookies

Reply:
Makes sense to get it from store state. Thanks for pointing out. Will do it in the evening.

Reply:
Apologies for the delay. Change checked in. Please review.